docs/guide/browser.md

```diff
@@ -551,6 +551,31 @@ References:
 - [WebdriverIO `element.doubleClick` API](https://webdriver.io/docs/api/element/doubleClick/)
 - [testing-library `dblClick` API](https://testing-library.com/docs/user-event/convenience/#dblClick)
 
+### userEvent.tripleClick
+
+- **Type:** `(element: Element, options?: UserEventTripleClickOptions) => Promise<void>`
+
+Triggers a triple click event on an element
```

Comment:
we should add here the user should check `evt.detail === 3`  since the browser will fire 3 clicks in a row